src/test/java/com/google/api/codegen/testdata/nodejs/nodejs_doc_test_library.baseline

```diff
@@ -1387,6 +1393,12 @@ describe('LibraryServiceClient', function() {
         done();
       });
     });
+
+    it('has longrunning decoder functions', function() {
+      var client = library.v1();
+      assert(client.longrunningDescriptors.getBigNothing.responseDecoder instanceof Function);
```

Comment:
Ha! Good job.

Reply:
Hehe. Hope we have an easier time catching it if theres a next time!